### PR TITLE
Update `create-rule` script

### PR DESCRIPTION
**Pull request by fisker**
- Remove `partly`
- Try open VSCode after rule created

### CODE REVIEW

File: scripts/create-rule.js [hunk 1]
```diff
@@ -199,6 +192,16 @@ function updateReadme(data) {
 	});
 	updateIndex(id);
 	updateReadme(data);
+
+	try {
+		await execa('code', [
```
Comment (sindresorhus) [inline scripts/create-rule.js:197]: I don't think you should assume VSCode. How about using https://github.com/sindresorhus/open-editor ?

Comment (fisker) [reply scripts/create-rule.js:197]: It seems open-editor  can't start my editor on windows, I'll look into it.

Comment (fisker) [reply scripts/create-rule.js:197]: Found: vscode can't use `--goto` when opening multiple files.

Comment (fisker) [reply scripts/create-rule.js:197]: https://github.com/sindresorhus/open-editor/issues/15

**Pull request merged**